pkg/fixer/fixer.go

```diff
@@ -0,0 +1,117 @@
+package fixer
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/open-policy-agent/opa/ast"
+	"github.com/open-policy-agent/opa/format"
+
+	"github.com/styrainc/regal/pkg/fixer/fixes"
+	"github.com/styrainc/regal/pkg/report"
+)
+
+func Fix(rep *report.Report, readers map[string]io.Reader, _ fixes.Options) (map[string][]byte, error) {
+	fixableViolations := map[string]struct{}{
+		"opa-fmt":     {},
+		"use-rego-v1": {},
+	}
```

Comment:
I think that something like what we have in OPA for builtins would be ideal here.

e.g.

```
var Builtins []*Builtin

// RegisterBuiltin adds a new built-in function to the registry.
func RegisterBuiltin(b *Builtin) {
	Builtins = append(Builtins, b)
	BuiltinMap[b.Name] = b
	if len(b.Infix) > 0 {
		BuiltinMap[b.Infix] = b
	}
}
```

then the fixes init functions can call RegisterFix or something?

Reply:
I guess it could just be a map of name -> func too?

Reply:
This is how it works now:

```go
	f := fixer.Fixer{}
	f.RegisterFixes(fixer.NewDefaultFixes()...)
```

I think this is ok too?